### PR TITLE
Upgrade to pac4j 1.8 + REST support + Authorizer + ProfileManager

### PR DESCRIPTION
**Pull request by leleuj**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,404 +1,386 @@
...
+- `play-pac4j-java` library (<em>groupId</em>: **org.pac4j**, *latest version*: **2.0.0-SNAPSHOT**) if you code in Java
+- `play-pac4j-scala_2.11` library (<em>groupId</em>: **org.pac4j**, *latest version*: **2.0.0-SNAPSHOT**) if you use Scala
 
-    class Application extends Controller with Security {
+as well as on the appropriate `pac4j` modules (<em>groupId</em>: **org.pac4j**, *version*: **1.8.0-SNAPSHOT**): the `pac4j-oauth` dependency for OAuth support, the `pac4j-cas` dependency for CAS support, the `pac4j-ldap` module for LDAP authentication, ...  
 
+As snapshot dependencies are only available in the [Sonatype snapshots repository](https://oss.sonatype.org/content/repositories/snapshots/org/pac4j/), this repository must be added in the Maven *pom.xml* file for example:
 
-### Define the supported clients
+    <repositories>
```
Comment (nateww) [inline README.md]: Play doesn't use Maven, but Ivy. Mentioning Maven XML files seems out-of-place here...

Comment (leleuj) [inline README.md]: You're completely right, I started this documentation by a copy/paste of this one: https://github.com/pac4j/j2e-pac4j/blob/master/README.md

I have the ambition to have really close implementations and even closer usages for all pac4j implementations (starting with J2E and Play). Will fix it.

Thx.

Comment (leleuj) [inline README.md]: Here it is: https://github.com/pac4j/play-pac4j/commit/7fcf149d37bca0b9e0306e15a850fe9accaa8fdd

**Comment by nateww** [comment]
Much nicer!

**Comment by leleuj** [comment]
Any **other** feedback will be appreciated as well ;-)

**Pull request merged**